gridpapers.dtx

```diff
@@ -250,7 +250,7 @@
 \RequirePackage{kvoptions}
 \RequirePackage{xcolor}
 \RequirePackage{tikz}
-\usetikzlibrary{patterns.meta,calc}
+\usetikzlibrary{calc}
```

Comment:
I see that `\usetikzlibrary{patterns.meta}` is on line 774 below. Why is that necessary?

Reply:
The old version of patterns.meta (~2015, which is what I have in TeXLive 2018) redefines some tikz-internal commands (specifically \pgf@declarepattern) to use macros that pgf doesn't know about. So the patterns other than `lightcones` which use pgfdeclarepatternformonly break.